Prox/Prox/AppDelegate.swift

```diff
@@ -23,9 +26,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // display
         window?.makeKeyAndVisible()
+
+        setupFirebase()
+
         return true
     }
 
+    private func setupFirebase() {
+        FIRApp.configure()
+
+        if let user = FIRAuth.auth()?.currentUser {
+            authorizedUser = user
+        } else {
+            FIRAuth.auth()?.signInAnonymously { (user, error) in
```

Comment:
Interesting – I only see [`signInAnonymouslyWithCompletion` in the docs](https://firebase.google.com/docs/reference/ios/firebaseauth/interface_f_i_r_auth.html#a0ffa43acd046e5ee42967106f73556f6) – am I looking at the right version of the docs?